lldb/include/lldb/API/SBDebugger.h

```diff
@@ -46,6 +47,7 @@ class LLDB_API SBDebugger {
       eBroadcastBitProgress = (1 << 0),
       eBroadcastBitWarning = (1 << 1),
       eBroadcastBitError = (1 << 2),
+      eBroadcastBitSymbolChange = (1 << 3),
```

Comment:
Sounds like this shouldn't be public per Jim's comments as the one from Debugger.h is designed to be private

Reply:
Yes, as per Jim's comments this shouldn't be exposed anymore and clients should use the `Target::eBroadcastBitSymbolsLoaded/Changed` bits instead. `Target::eBroadcastBitSymbolsChanged` is in the `Target` header file but isn't exposed through the SB API and isn't sent using an event broadcast.